docs/conf.py

```diff
@@ -213,9 +215,9 @@ def setup(app):
               'embedding': 'dwave-system',
```

Comment:
For Windows build with Python 3.10,  I'm hitting some problems:

* [setup_cfg['metadata']](https://github.com/dwavesystems/dwave-ocean-sdk/blob/c04a51752dfbe5e6f460752dc51617f15d620d7f/docs/conf.py#L30) gives a key error
* Line 219 `pkgs = [Distribution.from_name(req.name) for req in reqs]` gives `PackageNotFoundError: No package metadata was found for dwave-gate` that I think is because you switched `requires(extras=['all']` to `requires`

Reply:
Well, luckily, we're not building our docs on Windows :laughing: 

Can you make sure you ran `pip install .[all]` in your environment, just like we do in the CI job?

Note: `importlib.metadata.Distribution.requires` is now a property, hence it does not accept any parameters, but the code should work if the package is installed.

Reply:
Actually it turns out that if you run the above code line in the correct environment rather than some other random terminal, it in fact works.